presto-docs/src/main/sphinx/release/release-0.239.rst

```diff
@@ -0,0 +1,73 @@
+=============
+Release 0.239
+=============
+
+**Highlights**
+==============
+* Add support for ``DEFINER`` and ``INVOKER`` view security models.
+* Add support for caching Glue metastore in Hive connector.
+* Add Pinot SQL endpoint support.
+
+General Changes
+_______________
+* Fix :func:`classification_miss_rate` and :func:`classification_fall_out` (:pr:`14740`).
+* Fix ``NullPointerException`` in ``/v1/thread`` end point.
+* Fix an issue where the property ``ignore_stats_calculator_failures`` would not be honored
+  for certain queries that had filters pushed down to the table scan.
+* Fix missing query completion events for queries which fail prior to dispatching.
+* Fix potential infinite loop when the setting ``use_legacy_scheduler`` is set to ``false``.
+* Optimize cursor projection and filter by extracting and computing common subexpressions
+  among all projections and filter first. This can be disabled by the session property
+  ``optimize_common_sub_expressions``.
+* Add aggregation function :func:`set_union`.
+* Add local disk spilling support for queries with ``ORDER BY`` or ``DISTINCT``.
+* Add support to optimize min/max only metadata query. This is controlled by configuration
+  property ``optimizer.optimize-metadata-queries`` and session property ``optimize_metadata_queries``.
+  Note: Enabling this optimization might change query result if there are metadata that refers to
+  empty data, see :pr:`14845` for examples.
+* Add new unified grouped execution configuration property ``grouped-execution-enabled`` and
```

Comment:
Can you also mention here that ``grouped-execution-for-aggregation-enabled`` and ``grouped-execution-for-join-enabled`` will be removed in a future release.

Reply:
mentioned `grouped-execution-for-aggregation-enabled` next to the point below where I say that it is enabled by default. Added the point about join one here